docs/_docs/Instructors/02-Teachers-How-to-build-Learning-Units.md

```diff
@@ -151,18 +194,93 @@ tries to solve the Exercises.
 
 ### 5. Hackathon Problem Statement and Dataset
 
-> ⚠️ Section in Progress 
+The hackathon **dataset** and **problem statement** consist of the
+two deliverables that will be presented to students on the hackathon day, at the
+end of a specialization. They are released on the day of the hackathon and have the 
+format described below:
+
+* The dataset is a file or set of files provided directly in the repository containing
+what we consider training data. We typically work with small to medium size datasets to
+simplify the type of processing students need to do, given they work mostly on their own 
+computers. As such, you shouldn't need to exceed the size limitations that github imposes.
+* The problem statement consists of a small narrative and a set of instructions that 
+indicates to students what is the task at end - what they are trying to predict and some
+guidelines on how they are goign to be scored. This is typically shared through a public
+link to a google document, provided in the README of the hackathon folder
+
+Hackathons are a key moment in every specialization. They allow students to
+put the skills they learned throughout the units in practice while working 
+together with other students. Working on different datasets and problems
+gives students the opportunity to expand their knowledge beyond constrained 
+exercises, gaining practice that is as close as possible to a real life scenario.
+
+For this purpose, when preparing a hackathon, you should try to keep the following
+in mind:
+
+1. The dataset should be real (extracted from a web source, for example), not fabricated
+2. The problem should be as realistic as possible, this is, it could be something a data 
+scientist would do in real life
+3. The data and problem should get a decent baseline using at least part or all of the
+techniques taught in the specialization
+
+Finally, building the dataset and problem statement involves more than that. It assumes you
+already explored and tested out the dataset yourself (and potentially got more people to help) 
+so that you know what to expect from different techniques/splits/etc. This is something 
+that takes time, but is important to provide a solid hackathon, and a solid baseline solution (see
+ [Instructor Baseline Solution](#7-hackathon-instructor-baseline-solution) below)
+
+Expected time investment: 
+
+| Development | Instructor | Time (hours of investment) |
+|---|---|----|
+| Hackathon problem | Experienced instructor | 8 |
+| Hackathon problem | Junior instructor | 16 |
```

Comment:
I think the times here are a bit on the hopeful side of things. Finding a new dataset, creating a problem, creating the scoring function file and writing the prompt I think would take 10 to 12 hours for an experienced instructor and 20 to 24 hours for a junior instructor, assuming they are working alone, which sometimes is not the case. I don't know if it's worth mentioning this as well, so I'll leave this last suggestion up to you to include in the text or not. :)

Reply:
https://github.com/LDSSA/wiki/issues/364

Reply:
Agree with @cimendes that this is too optimistic, although I am quite slow on creating materials. 😅

But I think it depends mainly on the learning unit content, besides the experience level. As an example, it took me way more than 20 hours to create an entire exercise notebook on linear algebra, however for a simple unit like pandas 101 it would surely not take me more than 10. Some types of contents are naturally easier to come up with exercises and a dataset than others.

Reply:
Maybe that's also related to how comfortable you felt in each topic? 

Regardless, as mentioned these are placeholders to update - please check https://github.com/LDSSA/wiki/issues/364 - and if you haven't replied to the form please do 🙏